app/helpers/form_helper.rb

```diff
@@ -67,10 +67,8 @@ def multiple_selects(f, attr, associations, selected_ids, options = {}, html_opt
     end
     authorized = AssociationAuthorizer.authorized_associations(associations.reorder(nil), klass).all
 
-    # select2.js breaks the multiselects disabled items location
-    # http://projects.theforeman.org/issues/12028
     html_options["class"] ||= ""
-    html_options["class"] += " without_select2"
+    html_options["class"] += " jquery-multiselect"
```

Comment:
not sure I understand? without_select2 class avoids using select2, I was under the assumption you want to move from a black list (when someone adds the without_select2) to a whitelist of when we use select2?

Reply:
Yes, but I wanted to keep the form generating code the same so as to minimize changes in the application.  Basically I just want normal selects that I add with react to not automatically get the select2/multiselect classes.